blockchain/blockchain.go

```diff
@@ -472,7 +472,9 @@ func (bc *blockchain) commitBlock(blk *block.Block) error {
 		return err
 	}
 	blkHash := blk.HashBlock()
-	blk.HeaderLogger(log.L()).Info("Committed a block.", log.Hex("tipHash", blkHash[:]))
+	if blk.Height()%100 == 0 {
```

Comment:
https://github.com/uber-go/zap/blob/master/config.go#L39

Reply:
it is used to filter out excessive logs, say > 20 same entries per second

i don't think this fits our use-case? since we are reducing to a very small frequency, log 1 block out of 100. If we apply this sampling, means we would also only log 1 error out of 100.